openstack/orchestration/v1/stacks/template.go

```diff
@@ -98,14 +99,21 @@ func (t *Template) getFileContents(te interface{}, ignoreIf igFunc, recurse bool
 							if err := childTemplate.getFileContents(childTemplate.Parsed, ignoreIf, recurse); err != nil {
 								return err
 							}
+							childTemplate.fixFileRefs()
 						}
 					}
+					// If it doesn't parse as a template, it a 'get_file'
```

Comment:
I added that comment because initially I hadn't realized that other files than templates were also going through this code, despite the naming `childTemplate` and related comments. But because the parsing (as template) fails, no `getFileContents()` is executed on those. They are still inserted in `t.Files[]` though.
So I thought I'd try to clarify that. No so successful it seems. I'll try to rephrase.